django/db/models/sql/compiler.py

```diff
@@ -651,7 +651,9 @@ def find_ordering_name(self, name, opts, alias=None, default_order='ASC',
                                                        order, already_seen))
             return results
         targets, alias, _ = self.query.trim_joins(targets, joins, path)
-        return [(OrderBy(t.get_col(alias), descending=descending), False) for t in targets]
```

Comment:
I would appreciate some guidance here, @jarshwah @charettes (and anyone else..)

It's my understanding that all the members of `targets` and `final_field` are guaranteed to have the same value, so I don't understand why we're looping over targets and generating `OrderBy` objects. It seems to me that if there was a reason we needed to generate multiple `ORDER BY` clauses then the `final_field` should be part of this too. It seems that just using `final_field` and not `targets` produces correct results, but I fear there is a subtlety of joins that I'm missing.